components/src/atoms/ListItem/index.tsx

```diff
@@ -54,6 +63,7 @@ export function ListItem(props: ListItemProps): JSX.Element {
 
   const LIST_ITEM_STYLE = css`
     background-color: ${listItemProps.backgroundColor};
+    color: ${listItemProps.color};
```

Comment:
most cases color is `undefined` so probably passing value would be good.

```suggestion
    color: ${listItemProps.color ?? COLORS.black90};
```